### PR TITLE
Add linux to azure builds

### PR DESCRIPTION
**Pull request by jenshnielsen**

### CODE REVIEW

File: qcodes/tests/test_plot_utils.py [hunk 1]
```diff
@@ -3,7 +3,9 @@
 """
 
 from pytest import fixture
```
Comment (jenshnielsen) [inline qcodes/tests/test_plot_utils.py]: Not needed should be reverted

**Comment by jenshnielsen** [comment]
This seems to work but is way more ugly than I hoped 😞

**Comment by jenshnielsen** [comment]
@astafan8 All the steps that are the same are reused. It's just that many of the steps are only almost identical 😞

**Comment by astafan8** [comment]
@jenshnielsen shall we get this in?

i don't have anything against this as long as we are sure that the CI for the two platforms are executed in parallel (not sequentially).

**Comment by jenshnielsen** [comment]
They run parallelly but I think we should clean this up a bit more before merging.

**Comment by jenshnielsen** [comment]
Given the current issues with conda activation. I think this is as clean as its going to get until something is fixed externally. I added a comment to explain the current state

**Pull request merged**